cmd/erasure-object.go

```diff
@@ -1096,6 +1099,11 @@ func (er erasureObjects) PutObjectTags(ctx context.Context, bucket, object strin
 		if tags != "" {
 			fi.Metadata[xhttp.AmzObjectTagging] = tags
 		}
+		if opts.UserDefined != nil {
```

Comment:
not needed just the following line is enough.

```suggestion
		for k, v := range opts.UserDefined {
		}
```